### PR TITLE
Depend on version 1.8 of the pipeline

### PR DESCRIPTION
**Pull request by bertfrees**

### CODE REVIEW

**Comment by bertfrees** [comment]
@egli Try `mvn clean install`ing daisy/pipeline-scripts/pull/33 and `mvn clean package` this. All tests should pass. Haven't tried the .deb yet but I'm pretty confident it will work. Let me know if you're having trouble installing/getting it to work.

File: pom.xml [hunk 1]
```diff
@@ -20,270 +20,178 @@
...
-    <libreoffice-uno.version>4.0.2-SNAPSHOT</libreoffice-uno.version>
+    <!--
+	this needs to match the version in script-utils-aggregator
+    -->
+    <odt-utils.version>1.0.0</odt-utils.version>
     <tagNameFormat>v@{project.version}</tagNameFormat>
   </properties>
   
+  <!--
+      TODO: this belongs in modules-parent
```
Comment (egli) [inline pom.xml]: @bertfrees which `modules-parent` are you talking about? The one in sbsdev? If so why not just put it there?

Comment (bertfrees) [inline pom.xml]: I dunno, it was more a suggestion. It's up to you really. I didn't want to mess around with your parent pom in order not to break other projects. By putting this in the parent you can make sure all the child projects are compiled and tested against the same version of the pipeline, and you don't have to keep less version numbers in the child poms.

**Comment by egli** [comment]
I think it's fine to put it in the parent pom. @bertfrees can you do this? Thanks.

**Comment by bertfrees** [comment]
blocked on: daisy/pipeline-scripts/pull/33 and sbsdev/maven-parents/pull/1

**Pull request merged**